pkg/api/dashboard.go

```diff
@@ -748,13 +748,15 @@ func (hs *HTTPServer) GetDashboardVersion(c *models.ReqContext) response.Respons
 
 // swagger:route POST /dashboards/validate dashboards validateDashboard
 //
-// Validates a dashboard JSON against the schema
+// # Validates a dashboard JSON against the schema
```

Comment:
```suggestion
// Validates a dashboard JSON against the schema.
```

It's a little bit silly, but eh